specification/purview/data-plane/Azure.Analytics.Purview.Insights/preview/2022-03-01-preview/purviewinsights.json

```diff
@@ -0,0 +1,1202 @@
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SnapshotRecord": {
+      "required": ["kind"],
+      "type": "object",
+      "properties": {
+        "kind": {
+          "enum": [
+            "AssetFilesAggregation",
+            "AssetDistribution",
+            "FilesWithoutResourceSet",
+            "AllAssetsDistribution",
+            "AssetCurationDistribution",
+            "DeletedAssetsDistribution",
+            "NetNewAssetsDistribution",
+            "Tags"
+          ],
+          "type": "string"
+        },
+        "snapshotTime": {
+          "format": "date-time",
+          "type": "string"
+        }
+      },
+      "discriminator": "kind"
+    },
+    "Snapshot": {
+      "type": "object",
+      "properties": {
+        "snapshotTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SnapshotRecord"
+          }
+        },
+        "count": {
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "ErrorModel": {
+      "type": "object",
+      "properties": {
+        "errorCode": {
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "type": "string",
+          "readOnly": true
+        },
+        "exceptionCulprit": {
+          "type": "string",
+          "readOnly": true
+        },
+        "target": {
+          "type": "string",
+          "readOnly": true
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorModel"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "type": "object",
+      "properties": {
+        "error": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          ]
+        }
+      }
+    },
+    "DetailsRecord": {
+      "required": ["kind"],
+      "type": "object",
+      "properties": {
+        "kind": {
+          "enum": [
+            "AssetFilesAggregation",
+            "AssetDistribution",
+            "FilesWithoutResourceSet",
+            "AllAssetsDistribution",
+            "AssetCurationDistribution",
+            "DeletedAssetsDistribution",
+            "NetNewAssetsDistribution",
+            "Tags"
+          ],
+          "type": "string"
+        },
+        "snapshotTime": {
+          "format": "date-time",
+          "type": "string"
+        }
+      },
+      "discriminator": "kind"
+    },
+    "Details": {
+      "type": "object",
+      "properties": {
+        "snapshotTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DetailsRecord"
+          }
+        },
+        "count": {
```

Comment:
Using the same requirement as [Purview Scanning API](https://github.com/Azure/azure-rest-api-specs/blob/5dad3b83d0926768ede15ed41895d5f38defd512/specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2021-10-01-preview/scanningService.json#L2346). This API, however, is paginated by the client instead of a continuation token.